unit_tests/test_version_utils.py

```diff
@@ -608,6 +608,7 @@ def test_comparable_scylla_operator_versions_compare(version_string_left, versio
     ("1.9.0-alpha.1-nightly", "1.9.0-alpha.1"),
     ("1.9.0-alpha.1-2-g3321624", "1.9.0-alpha.1-2-g3321624"),
     ("1.9.0-alpha.1-2-g3321624-nightly", "1.9.0-alpha.1-2-g3321624"),
+    ("2024.2.0.dev.0.20231219.c7cdb16538f2.1", "2024.2.0-dev.0.20231219.c7cdb16538f2.1"),
```

Comment:
This is the `ComparableScyllaOperatorVersion` unit test, not the `ComparableScyllaVersion.